PolyDock/PolyDock/src/pd/ecs/sys/tabsHeader/TabsHeader.cpp

```diff
@@ -79,16 +79,26 @@ void TabsActivationSystem::update(entt::registry& registry, entt::entity root) c
 // ---------------------------------------------------------------------------------------------------------
 void TabsSelectionSystem::update(entt::registry& registry, entt::entity root) const
 {
-	if (auto* inputCmp = registry.try_get<root::Input>(root); inputCmp
-		&& inputCmp->wasJustPressed(root::Input::eMouseButton::LEFT))
+	if (auto* inputCmp = registry.try_get<root::Input>(root))
 	{
 		auto view = registry.view<tabsHeader::HoveredTab>();
 
 		for (auto entity : view)
 		{
-			registry.get_or_assign<tabsHeader::SelectedTabs>(entity).selectedTabs =
-			{ view.get<tabsHeader::HoveredTab>(entity).hoveredTab };
+			const auto& hoveredCmp = view.get<tabsHeader::HoveredTab>(entity);
```

Comment:
do we want to stick to a particular `using` convention?

Reply:
yes. After removing 'Component' from all component classes I came to a conclusion that since there are namespaces like tabsHeader and tabbedWindow there is no need to call widget components TabbedWindowComponent and TabsHeaderComponent. Now they are called just Widget and they are distinguishable by the namespace. And we have more components like this.